api/src/main/java/io/knotx/engine/api/KnotFlow.java

```diff
@@ -21,65 +21,60 @@
 import java.util.Map;
 import java.util.Objects;
 
-@DataObject
+@DataObject(generateConverter = true)
 public class KnotFlow {
```

Comment:
If I understand correctly, this class represents a step in a fragment processing flow.
Is this `KnotFlow` or rather `FragmentProcessingStep` or `FragmentFlowStep`?